lib/ama_layout/breadcrumb_builder.rb

```diff
@@ -0,0 +1,23 @@
+module AmaLayout
+  class BreadcrumbBuilder < BreadcrumbsOnRails::Breadcrumbs::Builder
+    def render
+      @elements.map { |e| render_element(e) }.join(@options[:separator])
+    end
+
+  private
+
+    def render_element(element)
+      name = compute_name(element)
+      path = element.path && compute_path(element) || '#'
```

Comment:
I used the code [here](https://github.com/weppos/breadcrumbs_on_rails/blob/master/lib/breadcrumbs_on_rails/breadcrumbs.rb#L79-L100) as an example.